packages/docs/src/scripts/components/PageHeader.jsx

```diff
@@ -26,6 +26,16 @@ class PageHeader extends React.PureComponent {
     }
   }
 
+  guidanceLink() {
+    if (this.props.showGuidanceLink) {
+      return (
+        <a className="ds-u-margin-right--2" href="#guidance">
+          View guidance
```

Comment:
I noticed the other two links are to external sites. I wonder if we want to distinguish this link more? like maybe changing the content to `Jump to guidance` or adding a spacer somehow? Do you have any thoughts on this @melwoodard?

Reply:
Updated to `jump to guidance`